### ISSUE TITLE
Failure to build 1.9.3

### ISSUE THREAD

**Issue by ArchangeGabriel** [go-gitea/gitea#8126]
- Gitea version (or commit ref): 1.9.3
- Operating system: ArchLinux (go 1.13)

## Description

Running `make vendor`:
```
go: github.com/go-macaron/cors@v0.0.0-20190309005821-6fd6a9bfe14e9: invalid pseudo-version: revision is longer than canonical (6fd6a9bfe14e)
```

**Comment by techknowlogick** [comment]
Sadly this is due to an upstream dependency, which prevents 1.9 building with golang 1.13. We have resolved this in 1.10 which will has started its release process.

**Comment by 6543** [comment]
@techknowlogick when 1.10.x ? because now https://github.com/alpinelinux/aports/pull/10887 fails and im still stuck with 1.8.3!
